+sig/Signal.m

```diff
@@ -507,6 +511,23 @@
       % input signal
       x = map(strSig, @str2num, 'str2num(%s)');
     end
+    
+    function b = round(a)
```

Comment:
@jaib1 Forgot to add this to the review but the reason I refrained from adding round is because I wanted to make sure that the optional args would be correctly implemented.  Could you redo this so that `round(X,N,type)` is possible in signals?  These are very useful and oft-used parameters

Reply:
Ah yup, sorry lazy on my part